src/Client/OrchestrationServiceClientShim/ShimDurableEntityClient.cs

```diff
@@ -90,7 +92,10 @@ public override async Task SignalEntityAsync(
             EntityMessageEvent.GetCappedScheduledTime(
                 DateTime.UtcNow,
                 this.options.Entities.MaxSignalDelayTimeOrDefault,
-                scheduledTime?.UtcDateTime));
+                scheduledTime?.UtcDateTime),
+            Activity.Current is { } activity ? new DistributedTraceContext(activity.Id!, activity.TraceStateString) : null,
+            DateTimeOffset.UtcNow,
+            createTrace: true);
```

Comment:
nit: can you indent these lines?

Reply:
Should they be indented? It looks like they're on the same level as all the other parameters for the `ClientEntityHelpers.EmitOperationSignal` call